client/lib/wpcom-undocumented/lib/site.js

```diff
@@ -195,6 +195,10 @@ UndocumentedSite.prototype.postCounts = function( options, callback ) {
 	return this.wpcom.req.get( '/sites/' + this._id + '/post-counts/' + type, query, callback );
 };
 
+UndocumentedSite.prototype.postTypeTaxonomies = function( postType, callback ) {
```

Comment:
>we would wait to merge this PR https://github.com/Automattic/wpcom.js/pull/193 and try to avoid using wpcom-undocumented as much as possible

Sure, I went ahead with the changes here despite Automattic/wpcom.js#193 to help me build tests while developing, and wasn't certain how much effort it'd take to get a new wpcom.js version published. Happy to hold off for that feature to be merged, though.